include/core/CStateDecompressor.h

```diff
@@ -11,17 +11,21 @@
 #ifndef INCLUDED_ml_core_CStateDecompressor_h
 #define INCLUDED_ml_core_CStateDecompressor_h
 
+#include <core/CBoostJsonUnbufferedIStreamWrapper.h>
 #include <core/CDataSearcher.h>
-#include <core/CRapidJsonUnbufferedIStreamWrapper.h>
 #include <core/ImportExport.h>
 
 #include <boost/iostreams/filtering_stream.hpp>
+#include <boost/json.hpp>
 
-#include <rapidjson/reader.h>
+namespace json = boost::json;
 
 namespace ml {
 namespace core {
 
+//! Maximum size of the buffer (in bytes) used to hold a compressed chunk
+static const int COMPRESSED_CHUNK_BUFFER_LENGTH = 1638400;
```

Comment:
```suggestion
static const std::size_t COMPRESSED_CHUNK_BUFFER_LENGTH = 1638400;
```

Also, should it be a class member rather than loose in the namespace?